source/tests/system/niscope_driver_api_tests.cpp

```diff
@@ -296,7 +296,7 @@ TEST_F(NiScopeDriverApiTest, NiScopeFetchArrayMeasurement_SendRequest_FetchCompl
   EXPECT_TRUE(status.ok());
   expect_api_success(response.status());
   EXPECT_EQ(expected_num_waveforms * expected_waveform_size, response.meas_wfm_size());
-  EXPECT_EQ(expected_waveform_size, response.wfm_info_size());
+  EXPECT_EQ(expected_num_waveforms, response.wfm_info_size());
```

Comment:
This looks reasonable to me but I would want some scope expert to sign off for this change. @marcoskirsch can you take a look at this?

Reply:
I _think_ this is correct.

Reply:
Sounds good. I'm pretty confident this is correct looking at the documentation and what nimi-python does. Without this change the corresponding nimi-python test using grpc-device failed on the assert about the wfm_info size and with this update it passes so I'll go ahead and submit this. I think we had it wrong on this side and no one had caught it yet.